packages/ifpack2/src/Ifpack2_AdditiveSchwarz_def.hpp

```diff
@@ -1041,6 +1046,7 @@ getValidParameters () const
     const bool filterSingletons     = false;
     const int  numIterations        = 1;
     const bool zeroStartingSolution = true;
+    const double updateDamping      = 1;
```

Comment:
You use `scalar_type` as the parameter's type above, but use `double` here.  This could cause failures in two cases:

  1. `scalar_type` is `complex<T>`
  2. `scalar_type` is `float`

Reply:
Yeah, that's a MueLu-style intentional inconsistency.